package.json

```diff
@@ -141,7 +141,7 @@
     "eslint-plugin-promise": "4.0.1",
     "expect.js": "0.3.1",
     "husky": "3.1.0",
-    "jest": "24.9.0",
+    "jest": "^29.7.0",
```

Comment:
**suggestion:** If we're bumping this, can we also bump `@types/jest` on L124? (Seems to have fallen quite behind 🥲)